#!/usr/bin/env python
# field of view configuration

PACKAGE="pcd_tutorial"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("leafsize", double_t, 1, "Leafsize", 0.02, 0.002, 0.3)
gen.add("MeanK", int_t, 1, "MeanK", 50, 1, 500)
gen.add("StddevMulThresh", double_t, 1, "StddevMulThresh", 0.5, 0.0, 10.0)

exit(gen.generate(PACKAGE, "pcd_tutorial", "voxel_filter_node"))
